queries.sql

```diff
@@ -1,3 +1,34 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/* Find all animals whose name ends in "mon". */
+
+SELECT * FROM animals WHERE name LIKE '%mon%';
+
```

Comment:
**Changes Required**
- [ ] This will not get animals ending with `mon` but rather containing `mon`. Let us please try and fix this.